packages/venia-ui/lib/components/CartPage/GiftCards/giftCard.css

```diff
@@ -1,4 +1,13 @@
 .card_info {
+    display: flex;
+    flex-direction: column;
+}
+
+.balance {
+    font-size: 0.875rem;
```

Comment:
This is computed based on the Venia Styleguide:

![Screen Shot 2020-02-10 at 11 52 40 AM](https://user-images.githubusercontent.com/13182778/74177460-5d2a0500-4bff-11ea-80a9-49e67aa37698.png)

Since we don't have CSS Properties (variables) for font sizes yet, `0.875rem` = `14px`.